src/GeositeFramework/js/Map.js

```diff
@@ -7,15 +7,17 @@ require(['use!Geosite',
          'esri/map',
          'esri/layers/ArcGISTiledMapServiceLayer',
          'esri/geometry/Extent',
-         'esri/SpatialReference'
+         'esri/SpatialReference',
+         'esri/dijit/Search',
```

Comment:
I think we have run into issues in the past with trailing commas in some versions of IE.